### PR TITLE
Radio Labels indicate which button is selected.

### PR DESCRIPTION
**Pull request by dwilbanks**
https://github.com/cakephp/cakephp/issues/10235

### CODE REVIEW

File: src/View/Widget/RadioWidget.php [hunk 1]
```diff
@@ -172,6 +172,7 @@ protected function _renderInput($val, $text, $data, $context)
         }
         if (isset($data['val']) && (string)$data['val'] === (string)$radio['value']) {
             $radio['checked'] = true;
+            $radio['templateVars']['activeClass'] = "active";
```
Comment (markstory) [inline src/View/Widget/RadioWidget.php]: Any chance a test could be added to ensure this templateVar is not lost in future refactorings?

**Review by markstory — CHANGES_REQUESTED** [review]

**Comment by ADmad** [comment]
Don't really like this. Why can't you just use `input:checked` selector?

**Comment by ADmad** [comment]
From your original issue I see you want it mainly to target the labels when using bootstrap. In that case I would prefer this change was done in [friendsofcake/bootstrap-ui](https://github.com/FriendsOfCake/bootstrap-ui) instead.

**Comment by dwilbanks** [comment]
Bootstrap is one example, but, should not be the only.  

The friendsofcake/bootstrap-ui class does not implement the _renderInput method is not being implemented by friendsofcake, they inherit from you.

If the change is implemented in friendsofcake, they will be required to re-implement this single line every time changes are made in cakephp\cakephp.

Don't you think that the label associated with the checked radio item should be allowed to style differently than all others?

**Comment by ADmad** [comment]
Fair point about having to override the whole `_renderInput()` method.

I am just concerned about adding new harcoded classes. But since this change is being made to `templateVars` is shouldn't affect default templates, so I guess it's relatively safe.

File: src/View/Widget/RadioWidget.php (hunk 1 above)
Comment (ADmad) [inline src/View/Widget/RadioWidget.php]: Please use single quotes for consistency.

Comment (dwilbanks) [inline src/View/Widget/RadioWidget.php]: oops.

**Pull request merged**